### PR TITLE
Fix for APIMANAGER-4006

### PR DESCRIPTION
**Pull request by thilinicooray**
Fix for Sample API creation fails when unlimited tier is not available

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/template.jag [hunk 1]
```diff
@@ -195,7 +199,7 @@
         <%if(outputs.isCreatePermitted){ %>
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/template.jag]: Formatting issue (space needed before curly brace)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js [hunk 2]
```diff
@@ -81,10 +81,7 @@ APISamples.prototype.deploySample1 = function(){
...
 } 
 
-
-$(document).ready(function () {
-	$("#deploy_sample1").click(function(){
-		var deployer = new APISamples();
-		deployer.deploySample1();
-	});
-});
+var deploySampleApi = function(defaultTier){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js]: Formatting issue (space needed before curly brace)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-function APISamples () {
-    this.sample1_swagger = "{\"paths\":{ \"/*\":{\"get\":{\"parameters\":[{\"description\":\"Name of the City\",\"name\":\"q\",\"type\":\"string\",\"required\":false,\"in\":\"query\"}],\"responses\":{\"200\":{ } },\"x-auth-type\":\"Application & Application User\", \"x-throttling-tier\":\"Unlimited\" }}},\"swagger\":\"2.0\",\"info\":{ \"title\":\"WeatherAPI\",\"version\":\"1.0.0\"}}";
+function APISamples (defaultTier) {
+    this.sample1_swagger = "{\"paths\":{ \"/*\":{\"get\":{\"parameters\":[{\"description\":\"Name of the City\",\"name\":\"q\",\"type\":\"string\",\"required\":false,\"in\":\"query\"}],\"responses\":{\"200\":{ } },\"x-auth-type\":\"Application & Application User\", \"x-throttling-tier\":\"" + defaultTier + "\" }}},\"swagger\":\"2.0\",\"info\":{ \"title\":\"WeatherAPI\",\"version\":\"1.0.0\"}}";
 }
 
 
-APISamples.prototype.deploySample1 = function(){
+APISamples.prototype.deploySample1 = function(defaultTier){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js]: Some Formatting issues in these two lines.  (space needed before curly brace)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js (hunk 2 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js]: Please note that formatting issue has been corrected in current source [1].
[1] https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js#L124

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js (hunk 3 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js]: Please note that formatting issue has been corrected in the current source [1].
[1] https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js#L25

**Pull request merged**